anaUltraLatency.py

```diff
@@ -83,4 +83,3 @@
 
 
```

Comment:
Looks like you still haven't added the 8x3 summary plot yet, so I will wait to approve.